src/main/java/com/slobodianyk/java_spring_boot/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,22 @@
+package com.slobodianyk.java_spring_boot;
+
+import jakarta.persistence.EntityNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(EntityNotFoundException.class)
+    public ResponseEntity<String> handleNotFound(EntityNotFoundException ex) {
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(ex.getMessage());
+    }
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<String> handleValidationException(MethodArgumentNotValidException ex) {
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Validation error");
```

Comment:
"Validation error" - not meaningful error... It will be hard to understand what is wrong on the UI side.
Also, it is better to log the error for better debugging

```suggestion
        log.error("Validation error", ex)
        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Validation error");
```